utils/poa.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import sdkmath "cosmossdk.io/math"
+
+// WARNING: DO NOT MODIFY
+
+// Validators are paid 15% of revenue
+var PoaValPaymentRate = sdkmath.LegacyMustNewDecFromStr("0.15")
+
+// Hub Addresses
+// - cosmosvalcons1m7fg8k39k2tyym5hgwrpf5wx9hqsr8vywuyrtm
+// - cosmosvalcons1c5e86exd7jsyhcfqdejltdsagjfrvv8xv22368
+// - cosmosvalcons1pdpwglc4fcjdzqvyhvfwxg684trpc6uqck5sxk
+// - cosmosvalcons1px0zkz2cxvc6lh34uhafveea9jnaagckmrlsye
+// - cosmosvalcons1vz42ewp04wwepjed7z4qenj925gpakgvap4q2u
+// - cosmosvalcons1upc05nc9pwhhagnkr3f2dft327qxsxfeyvajsu
+// - cosmosvalcons1f6cjsfn47ujttypx7gtncglsmjvndugc2zelqx
+
+var PoaValidatorSet = []string{
+	"stride1gp957czryfgyvxwn3tfnyy2f0t9g2p4pxxdj7c",
```

Comment:
Should we validate these are real addresses in code? I assume you checked them manually on mintscan

Reply:
they're technically validated here: https://github.com/Stride-Labs/stride/pull/1456/files#diff-47c484455a6d234bf341de50b1221206d9e4a1acc603055c0096612bee25e8b0R67

and tested that that doesn't panic with the k8s net